demos/qadi_experiments/qadi_cmd.py

```
@@ -0,0 +1,151 @@
+        else:
+            print(" ❌ No response")
+            return
+    except Exception as e:
+        print(f" ❌ {str(e)}")
+        return
+    
+    # Run QADI with reduced complexity for faster response
+    print("\nRunning QADI analysis (this may take 30-60 seconds)...")
+    print("Progress:", end='', flush=True)
+    
+    start_time = time.time()
```

Ruff (0.11.9)

Local variable `start_time` is assigned to but never used

Remove assignment to unused variable `start_time`

(F841)